.github/workflows/monorepo-actor-release.md

```diff
@@ -0,0 +1,23 @@
+# Using the Monorepo Release Workflow
+
+## Setup
+
+Copy this workflow into your `.github/workflows` directory at the root of your project.
+
+Actors should be created within an `actor` directory at the root of your project.
+
+A GitHub repository or environment secret must be created for each actor in the format `WASH_$ACTORNAME_SECRET`.
+
+## Initiation
+
+The workflow is initiated by pushing a tag in the format `$actorname-vx.x.x` which must be the same version as the one in your actors cargo.toml file.
+
```

Comment:
Add note that if using templates to create actor, must delete the `.git` dir in actor.